docs/changelog.md

```
@@ -5,6 +5,14 @@
 sidebar_position: 9
 ---
 
+## October 31, 2024
+
+### Learnings and Metrics for CodeRabbit Self-Hosted Customers
+Self-Hosted CodeRabbit customers can now access Learnings and Metrics to gain deeper insights into their usage and performance. Through the CodeRabbit UI, customers can view metrics, track learnings, and easily generate an API key.
+To enable these features:
+- Generate an API key in the CodeRabbit UI. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)